src/realm/table_view.cpp

```diff
@@ -122,6 +122,10 @@ R ConstTableView::aggregate(ColKey column_key, size_t* result_count, ObjKey* ret
 
     typename Aggregator<T, action>::AggType agg;
     ObjKey last_accumulated_key = null_key;
+    if constexpr (!(action == act_Min || action == act_Max)) {
```

Comment:
I think it would be nicer to have this as an 'else' part of the 'if constexpr' below.